### PR TITLE
[11_15] Unit test to confirm bold and small-caps sys-chinese font resolves ok at first attempt

### PR DESCRIPTION
**Pull request by darcy-shen**

### CODE REVIEW

File: xmake/packages.lua [hunk 1]
```diff
@@ -71,13 +88,7 @@ function add_requires_of_mogan()
...
         add_requires("libiconv "..LIBICONV_VERSION, {system=false})
         add_requires("freetype "..FREETYPE_VERSION, {system=false})
-    end
-
-    if is_plat("mingw") or is_plat("windows") then
-        add_requires("qt6widgets "..QT6_VERSION)
-        if is_mode("release") then
-            add_requires("qtifw "..QTIFW_VERSION)
-        end
+        add_requireconfs("pdfhummus.freetype", {version = FREETYPE_VERSION, system = false, override=true})
```
Comment (jingkaimori) [inline xmake/packages.lua:91]: why this line is not grouped with other require configs of `pdfhummus`?

**Review by jingkaimori — COMMENTED** [review]
Generally no problem.

**Comment by jingkaimori** [comment]
maybe no chinese font is installed on archlinux image

File: xmake/packages.lua (hunk 1 above)
Comment (darcy-shen) [reply xmake/packages.lua:91]: it works, nevermind

And because there are other customizations per package.

Comment (darcy-shen) [reply xmake/packages.lua:91]: see https://github.com/XmacsLabs/mogan/pull/1465

**Pull request merged**